### ISSUE TITLE
Trying to scroll a Mintupdate window scrolls the value of a field

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/mintupdate#881]
**Describe the bug**
Trying to scroll a mintUpdate window scrolls the value of a field, if (and only if) the pointer is over a scrollable field. The user does not expect this. She expects that behaviour only if she has _clicked_  the field. The current behaviour is throwing and irritating and I think it was reported years ago but I cannot find an existing bug report. The bug is not, I am fairly sure, confined to mintUpdate.

**Screenshots**
![image](https://github.com/user-attachments/assets/5400e6d7-a511-4198-ba85-673128899caf)

In the screenshot above, a field shows as selected. I had not selected it. Using the mouse wheel scrolls the _value_ as against the window.

**To Reproduce**
1. Open mintUpdate.
2. Open its 'preferences'.
3. Scroll down and up the list.
4. Find that one is no longer scrolling up and down the list but instead inadvertently adjusting some value(s).

**Expected behavior**
The wheel should alter fields only after the field has been selected - selected via _a click_.

**Distribution:**
- [X] Linux Mint
- [ ] LMDE
- [ ] Other (please specify)

**Software version:**
6.0.9.

**Additional context**
This problem irritates considerably and, as I said, probably applies beyond the Update Mananger.

**Comment by JosephMcc** [comment]
This is far out of scope of Linux Mint. It's just how it works. If you are over a scrollable area, and the window has focus, the mouse wheel will scroll that area. It would actually be far more annoying if it stopped working that way after doing so forever. No one has reported this as a bug because it isn't one.

**Comment by LinuxOnTheDesktop** [comment]
> It would actually be far more annoying if it stopped working that way after doing so forever. 

One does not want to thwart habituated expectations. Please consider though the following.

1) Someone as yet unhabituated will be thrown, and might make changes she does intend.

Indeed - if this is a separate point -

2) it can confuse that wheel actions starts by scrolling a window and ends by changing a value.

1 and 2 suggest that the current behaviour is not one that should be used in a design. But since, as you say, it has long been used in Mint's design . .

**Comment by JosephMcc** [comment]
It's not "Mint" design. Those fields are just scrollable in gtk. Don't have any qt apps installed but it likely behaves the same there as well. Closing this since it isn't a bug.
